src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = "White";
+    private static final int DEFAULT_BOND = 100;
+    private static final int DEFAULT_RADIUS = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public enum FigureList {
+        CIRCLE,
+        ISOSCELES_TRAPEZOID,
+        RECTANGLE,
+        RIGHT_TRIANGLE,
+        SQUARE
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(FigureSupplier.DEFAULT_COLOR, FigureSupplier.DEFAULT_RADIUS);

```

Comment:
There is no need to indicate own class
```suggestion
        return new Circle(DEFAULT_COLOR, DEFAULT_RADIUS);

```